console-subscriber/src/aggregator/mod.rs

```diff
@@ -234,6 +236,15 @@ impl Aggregator {
                     }
                 };
             }
+            tracing::debug!(
+                async_resource_op = counts.async_resource_op,
+                metadata = counts.metadata,
+                poll_op = counts.poll_op,
+                resource = counts.resource,
+                spawn = counts.spawn,
+                total = counts.total(),
```

Comment:
tiny style nit: since these are counts, i would either pluralize the field names, like this:
```suggestion
                async_resource_ops = counts.async_resource_op,
                metadatas = counts.metadata,
                poll_ops = counts.poll_op,
                resources = counts.resource,
                spawns = counts.spawn,
                total = counts.total(),
```
or just leave the `counts.` prefix, like this:
```suggestion
                counts.async_resource_op,
                counts.metadata,
                counts.poll_op,
                counts.resource,
                counts.spawn,
                total = counts.total(),
```

up to you! :)

Reply:
Taken the first suggestion.